easy_run.py

```diff
@@ -149,7 +149,7 @@ def transfer_assignments_to_todoist():
                 print("Assignment already synced: " + assignment['name'])
                 is_added = True
                 # print(assignment)
-                if (task['due'] and task['due'].get('date') != assignment.get('due_date')):
+                if (task['due'] and task['due']['date'] != assignment['due_at']):
```

Comment:
Instead of allowing errors to be thrown. Adding an and statement to the if may be more stable.

`if(task['due'] and assignment.get('due_at') is not None and task['due'].get('date') is not None and task['due']['date'] != assignment['due_date']`

Reply:
Got it, hypothetically no errors would be thrown whatsoever unless either Canvas or Todoist would change their API.
However, if this happens then the application would just fail silently and cause confusion about what is going on.
Also, I am afraid checking against None would causes conflicts in the scenario in which an assignment's due date is added or removed from the assignment.

Would a better solution be to catch a KeyError and print a warning statement to console? Let me know your thoughts and I'll make the appropriate changes!
Thank you so much for this awesome tool! :)

Reply:
Ah, I see makes sense! As it is should suffice, I'll approve and merge. Thanks 🎉